make it respond

johngruen opened the pull request:
Fixes #41

pdehaan commented on src/preset-themes/006.json:
```diff
@@ -0,0 +1,44 @@
+{
```
Add: `"title": "006",`

pdehaan commented on src/preset-themes/default.json:
```diff
@@ -32,6 +32,11 @@
+    "tab_line": {
+      "h": 360,
+      "s": 100,
+      "l": 60
```
You'll need to tweak the /docs/theme-schema.json file as well. Along with all the other hsl[a] definitions, add the following new property:

```js
                "tab_line": {
                    "type": "object",
                    "properties": {
                        "h": {
                            "type": "integer",
                            "maximum": 360,
                            "minimum": 0
                        },
                        "s": {
                            "type": "integer",
                            "maximum": 100,
                            "minimum": 0
                        },
                        "l": {
                            "type": "integer",
                            "maximum": 100,
                            "minimum": 0
                        }
                    },
                    "required": ["h", "s", "l"],
                    "additionalProperties": false
                }
```

It's a bit of a funny one, since we've only added a `tab_line` definition to 2 of the 8 themes, so it isn't a required property unless we backfill it in to the other themes. But this should be enough to pass the super awesome theme linter.

johngruen commented:
Oh yeah,  i didn't see the new linter stuff after i rebased. looks awesome. I added tab_lines to all the current present themes (which are all going to change anyway)

pdehaan commented:
There is some cropping of the color pickers at 640 where the panel floats slightly off screen. Not sure if that is fixable, or even worth fixing. Stuff doesn't start _breaking_ until around 620px wide, where the "B" text input disappears off the right edge.

![themer640](https://user-images.githubusercontent.com/557895/37724181-6af17368-2ced-11e8-9785-efff8bb2cf1b.png)

Similar stuff happens (to lesser degree) for the "Toolbar Icons and Text" color picker on the left side of the UI, were the "Hex" text input and color chips float slightly off the edge, but is still usable:

![firefox_themer_640b](https://user-images.githubusercontent.com/557895/37724279-9ea5bd5e-2ced-11e8-86a3-29985047e276.png)

Probably the most problematic is the Theme Texture picker, since it causes some horizontal scrollage.

![firefox_themer_theme_texture](https://user-images.githubusercontent.com/557895/37724432-feec4e3a-2ced-11e8-9298-63777043b71b.png)

![firefox_themer_-dual_scrollage](https://user-images.githubusercontent.com/557895/37724552-44c7fbb6-2cee-11e8-8994-a2e273499f88.png)

johngruen commented:
@pdehaan yeah...maybe follow up bugs?
